apps/music/views/album-detail/view.js

```diff
@@ -34,8 +34,8 @@ AlbumDetailView.prototype.destroy = function() {
 };
 
 AlbumDetailView.prototype.render = function() {
-  View.prototype.render.call(this); // super();
-  this.list.setModel(this.songs);
+  return this.list.setModel(this.songs)
+    .then(View.prototype.render.bind(this)); // super();
```

Comment:
I wonder if we shouldn't just add an `onRenderDone()` method to the base `View` class instead? It seems odd to me to have a `super()` call occur async like this.